### ISSUE TITLE
Multiple images with >30 images

### ISSUE THREAD

**Issue by caohothang** [hellsan631/LogosDistort#15]
Can you test with >30 images, you will see problem, some images from 25-30, will broken...
My web
http://prntscr.com/8mtbje

Your demo 5
http://prntscr.com/8mtbrq

I will update my changes to git
Please test with 30 images and remove overflow hidden in body (css)

**Comment by hellsan631** [comment]
I can tell that the clipping is caused by not having a high enough `outerBuffer` and can be fixed by increasing that and/or changing the `perspectiveMulti`, or decreaing the `effectWeight`. I will make some small updates to performance.

We want overflow hidden on the css elements. This is important to the "looking through a window" effect and disabling it is not supported

**Comment by caohothang** [comment]
Thank you!
When you update please comment in here.

**Comment by hellsan631** [comment]
Ah I see what you meant by `overflow:hidden;`. because that was on `body`. You can take that off. See the new _demo5.html_ on the **master branch**. In general things should be much better. 

You will still see a lot of performance problems when there are a ton of images on screen, and if all of these images use very large file sizes with high resolutions. This isn't a problem I can easily solve, as it has to deal with browser performance with matrix3d transforms, and limits in GPU memory (so far, from what I've observed). However, all of these images would need to be in viewport at the same time.

Basically, if your running into this problem. Reduce the size of your images your loading (both in dimensions and kb), or don't load that many images on screen at once (make the containers bigger). Or, use one of the other mouse effects, not `'window'` (`'magnetic'` for example).

Post here and let me know if that solves everything. If there is still a problem, i'll reopen this issue

**Comment by caohothang** [comment]
I just tested with file demo5.html when i pulled code 
It only working with image 1 - 9

Image 10-over
not working when i hover mouse
http://prntscr.com/8n6sxg

Also it doesn't work in firefox

**Comment by hellsan631** [comment]
Looks like something happened and the events aren't firing correctly. Fixed some issues with the bound box, and it looks like this is somewhat related to #14

**Comment by hellsan631** [comment]
Let me know how the new changes work out.

The demo5.html has 108 different elements it applies distortions to. Its a bit extreme right now for performance testing, and I totally don't recommend using that amount, as the GPU memory is maxed out and it can lag badly.

**Comment by caohothang** [comment]
Thanks i will check it
